packages/react-native-reanimated/src/css/managers/CSSTransitionsManager.web.ts

```diff
@@ -11,6 +11,7 @@ import { kebabizeCamelCase } from '../utils';
 
 export default class CSSTransitionsManager implements ICSSTransitionsManager {
   private readonly element: ReanimatedHTMLElement;
+  private isAttached = false;
```

Comment:
How about the more private kind of private?
```suggestion
  #isAttached = false;
```